package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "opentelemetry",
-  "version": "0.24.0",
+  "version": "0.1.0",
```

Comment:
Was this intentional or did lerna edit this when you changed version to `independent`?

Reply:
I did it intentionally to make it clear this version has no meaning anymore. NPM gets mad if you remove it entirely.